client-v2/src/reducers/frontsReducer.js

```diff
@@ -17,34 +21,38 @@ type State = {
   }
 };
 
-const frontsConfig = (
+const reducer = (
   state?: State = {
-    frontsConfig: { fronts: {}, collections: {} },
+    frontsConfig: initialState,
     lastPressed: {
       draft: {},
       live: {}
     }
   },
   action: Action
 ): State => {
+  const frontsConfig = frontsConfigReducer(state.frontsConfig, action);
```

Comment:
Is there any reason this is nested inside this reducer? It seems like it could live outside but maybe I'm missing something?

Reply:
The frontsConfig operates on part of the fronts state as a nested reducer, so it needs to operate on the state passed in to the fronts reducer.

Reply:
I'll chat to you tomorrow about why it's nested. I don't see where the parent reducer touches frontsConfig so it looks a bit tucked away there but I'm sure it's well considered and again, I'm only questioning it because it's not how I would do it - not because it's not a good way of doing it!